### ISSUE TITLE
🐛 [Bug Report] [Classic]rogue spell blind in combat had so many miss

### ISSUE THREAD

**Issue by yyhhrr** [cmangos/issues#3517]
### Bug Details

Spell miss very much compared to other skills, I'm at 9% hit rate but in a fight the skill will fail to hit the target more than 4 out of 5 times, which I don't consider normal

### Steps to Reproduce

1.use blind on mobs
2.many unhit
3.
4.

### Core SHA1 Commit Hash

a03e728b38a689d47af00d2028404a5246159974

### Database SHA1 Commit Hash

c6a92ebd20b29abf3d84b186e187627fc8e46763

### Operating System

win 10

### Client Version

1.12.1 (Classic)

**Comment by kepheuswow** [comment]
Cannot confirm this issue. Used blind multiple times with my Rogue (14% Hit Chance).

CMaNGOS Classic
Core: d2d6631d3824fa7c2c3f28e525d33d2fedf2b510
DB: 1abab1db3866652080ed356b2039f498dee951e5

**Comment by yyhhrr** [comment]
yes,this issue disapear at 60level,but in low level maybe50,still very high miss
